Makefile

```diff
@@ -115,8 +115,8 @@ build-kind-images: build-kind-images-ko
 install: WHAT ?= ./cmd/...
 install:
 	GOOS=$(OS) GOARCH=$(ARCH) go install -ldflags="$(LDFLAGS)" $(WHAT)
-	ln -sfr $(INSTALL_GOBIN)/kubectl-workspace $(INSTALL_GOBIN)/kubectl-ws
-	ln -sfr $(INSTALL_GOBIN)/kubectl-workspace $(INSTALL_GOBIN)/kubectl-workspaces
+	ln -sf $(INSTALL_GOBIN)/kubectl-workspace $(INSTALL_GOBIN)/kubectl-ws
```

Comment:
`--relative` creates a symbolic link in linux. We could have separate commands depending on the OS, but wondering if its required here. Can we leave them to be absolute?

Reply:
Thanks for the fix - an absolute path should be fine I think - I used `-r` thinking we may want the symlinks to remain functional when copied in the Dockerfile, but we ended up runing `ln` [in the Dockerfile](https://github.com/kcp-dev/kcp/blob/main/Dockerfile#L53..L54) instead